perllib/FixMyStreet/App/Form/Waste/Garden.pm

```diff
@@ -251,6 +252,7 @@ has_field continue_review => (
 
 has_field submit => (
     type => 'Submit',
+    value => 'Continue to payment',
```

Comment:
Is this needed, your code change doesn't seem to affect this always being set above?

Reply:
I don't understand, it was removed then added according to the git log but isn't present in the files view. But I think can be removed!